src/components/App/index.tsx

```diff
@@ -41,8 +42,9 @@ class App extends React.Component<IProps> {
   private renderRoutes = () => {
     return (
       <Switch>
-        <Route exact path={`${routes.Passports}/:passportFactoryAddress?`} component={PassportsPage} />
-        <Route exact path={`${routes.PassportChanges}/:passportAddress?`} component={PassportChangesPage} />
+        <Route exact path={passportsPath} component={PassportsPage} />
+        <Route exact path={`${routes.LegacyPassportChanges}*`} component={PassportChangesRedirect} />
```

Comment:
Is there a need to have a "Route" here? We could use "PassportChangedRedirect" here immediately since it already checks fo LeagacyPassportChanges path

Reply:
i have no doubts regarding this, but//>